public/ticTacToe.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Tic-Tac-Toe</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="css/ticTacToe.css">
```

Comment:
It's beautiful. There is no x or o appearing, only light blue or dark blue. Don't know if its intentional, but its hard to tell as a user right away.